readme.md

```diff
@@ -18,52 +18,174 @@ For API reference, please see the
 [H3 Documentation](https://h3geo.org/).
 
 **NOTE: This version (v.3.6.1) corresponds to the new Cython
-bindings, which will replace the old bindings (v3.4.3).
-This version is not yet published to PyPI.**
+bindings, which have replaced the old bindings (v3.4.3).**
```

Comment:
Nit: The `v.3.6.1` here is going to get out of date pretty quickly. I'd change to something like:
```
Note: As of v3.6.1, the `h3-py` uses Cython as the underlying binding framework, replacing the older bindings in v3.4.x.
```

Reply:
Actually, we probably don't even need that bit. I just took it out.